projects/vdk-core/tests/vdk/internal/builtin_plugins/config/test_job_config.py

```diff
@@ -105,6 +105,35 @@ def test_vdk_options(self):
         cfg = JobConfig(self._test_dir)
         self.assertEqual({"a": "b"}, cfg.get_vdk_options())
 
+    def test_get_vdk_subsection_names(self):
+        self._write_to_config_ini(
+            self._test_dir,
+            """
+            [vdk_subsection_one]
+
+            [vdk_subsection_two]
+        """,
+        )
+        cfg = JobConfig(self._test_dir)
+        self.assertEqual(
+            list({"vdk_subsection_one", "vdk_subsection_two"}),
+            cfg.get_vdk_subsection_names(),
+        )
+
+    def test_vdk_subsection_values(self):
```

Comment:
Can you please change this test so it has a [vdk] section as well. We want to make sure that the standard configs get fetched correctly as well.

Reply:
@DeltaMichael the standard configs are fetched with another method called get_vdk_options and it is tested beforehand.